### PR TITLE
Implement From<&str> on Atom

### LINKED ISSUE #120: Missing clipboard constants
As far as I can tell, this crate is missing bindings to the `GDK_SELECTION_CLIPBOARD`, `GDK_SELECTION_PRIMARY`, and `GDK_SELECTION_SECONDARY` constants. These constants are necessary to implement cut/copy/paste with GTK.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes  #120.

### CODE REVIEW

File: src/atom.rs [hunk 1]
```diff
@@ -36,3 +36,9 @@ impl FromGlibPtr<ffi::GdkAtom> for Atom {
     #[inline]
     unsafe fn from_glib_full(_: ffi::GdkAtom) -> Atom { unimplemented!() }
 }
+
+impl<'a> From<&'a str> for Atom {
+    fn from(r: &'a str) -> Atom {
+        Atom::intern(r)
```
Comment (EPashkin) [inline src/atom.rs]: skip_assert_initialized!(); ?

Comment (GuillaumeGomez) [inline src/atom.rs]: Don't we want it to be checked?

Comment (gkoz) [inline src/atom.rs]: I suppose since `intern` checks that, it can be skipped here.

**Comment by gkoz** [comment]
Oh and it seems `appveyor.yml` still needs updating in this repo ;p

**Comment by GuillaumeGomez** [comment]
@gkoz: You want a commit in the same PR or I create another one?

**Comment by gkoz** [comment]
This PR is fine.

**Comment by GuillaumeGomez** [comment]
@gkoz: Merge?

**Comment by gkoz** [comment]
Thanks!

**Pull request merged**